assets/lang/es-es.cfg

```diff
@@ -4,16 +4,16 @@ Localization
 	{
 		astrogator_mainTitle = Astrogador
 		astrogator_versionFormat = v<<1>>.<<2>>.<<3>>
-		astrogator_mainTooltip = Sumario de transferencias a los cuerpos accesibles
+		astrogator_mainTooltip = Sumario de ventanas de transferencias a los cuerpos accesibles
 		astrogator_normalSubtitle = Transferencias desde <<1>>
 		astrogator_inboundHyperbolicWarning = <<1>> está en una trayectoria de escape.\nCaptura para ver más transferencias.
 		astrogator_outboundHyperbolicError = <<1>> está en una trayectoria de escape.\nCaptura para ver transferencias.
 		astrogator_landedError = <<1>> está aterrizado. Ponte en órbita para ver las transferencias.
 		astrogator_highInclinationError = La inclinación es <<1>>°, la precisión es demasiado baja pasados los <<2>>°
 		astrogator_noTransfersError = No hay transferencias disponibles
 		astrogator_launchSubtitle = Transferencias desde <<1>>\n(Lanzamiento ~<<2>>)
-		astrogator_transferColumnHeader = Cuerpo
-		astrogator_timeColumnHeader = Tiempo hasta ventana de transferencia
+		astrogator_transferColumnHeader = Transferir
```

Comment:
Thanks for the review!

That string is "Transfer" in English, so I'm inclined to use something similar here. It might not always be a simple list of celestial bodies; it's possible that things like "Jool via Eve" could be added someday.

If you had to explain to someone how to perform a "Hohmann transfer" in Spanish, would you say "Hohmann transferencia" in that case?